static/app/components/events/eventViewHierarchy.tsx

```diff
@@ -0,0 +1,86 @@
+import {useState} from 'react';
+import isEqual from 'lodash/isEqual';
+
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {tn} from 'sentry/locale';
+import {EventAttachment} from 'sentry/types';
+import {uniqueId} from 'sentry/utils/guid';
+import {useQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import {RenderingSystem} from './viewHierarchy/renderingSystem';
+import {ViewHierarchyTree} from './viewHierarchy/tree';
+import EventDataSection from './eventDataSection';
+
+const DEFAULT_RESPONSE = {rendering_system: '', windows: []};
+const FIVE_SECONDS_IN_MS = 5 * 1000;
+
+function fillWithUniqueIds(hierarchy) {
+  return {
+    ...hierarchy,
+    id: uniqueId(),
+    children: hierarchy.children.map(fillWithUniqueIds),
+  };
+}
+
+type Props = {
+  viewHierarchies: EventAttachment[];
+};
+
+function EventViewHierarchy({viewHierarchies}: Props) {
+  const [selectedViewHierarchy] = useState(0);
+  const [selectedViewHierarchyWindow] = useState(0);
```

Comment:
It was recommended that I future-proof for multiple view hierarchies so I've added a state variable to track which one is being viewed. I plan to follow up about this in another PR in case I need to add UI for selections.

There's also the possibility that there are multiple "windows" (e.g. a modal may take up a separate window). For now I've hard-coded 0 and plan to handle that in another PR as well.